pkg/master/master_test.go

```diff
@@ -256,6 +256,54 @@ func TestGetNodeAddresses(t *testing.T) {
 	assert.Equal([]string{"127.0.0.2", "127.0.0.2"}, addrs)
 }
 
+// Because we need to be backwards compatible with release 1.1, at endpoints
+// that exist in release 1.1, the responses should have empty APIVersion.
+func TestAPIVersionOfDiscoveryEndpoints(t *testing.T) {
```

Comment:
Yes. Before this PR, all discovery objects returned by server has "apiVersion" field equal to "v1".